### PR TITLE
Backend/database/mock data

### PR DESCRIPTION
**Pull request by barreryu**
Mock data and updated python scrip to create database
When script is run, will prompt with 3 choices
"Enter 1 to choose to create database"
"Enter 2 to enter in mock data"
"Enter 3 to do both"

### CODE REVIEW

File: database/createRecreateDatabase.py [hunk 1]
```diff
@@ -6,32 +6,50 @@
...
+import argparse
 
 # take environment variables from .env.
 load_dotenv()
+# Example URL to connect to: postgresql://ryubarrett:password@localhost:5432/opmdatabase
+database_url = os.getenv('DATABASE_URL')
 
 # File should be in same directory as python script
-CREATION_SQL_FILE = "RecreateOPMDatabase.sql"
+CREATION_SQL_FILE = "database\RecreateOPMDatabase.sql"
```
Comment (GPropersi) [inline database/createRecreateDatabase.py:17]: This assumes there's another `database` directory within database directory, i.e.

`/database/database/RecreateOPMDatabase.sql`

Could you remove the `database\` from these two?

File: database/MockDataOPMDatabase.sql [hunk 2]
```diff
@@ -0,0 +1,47 @@
+-- queries to add in mock data
```
Comment (GPropersi) [inline database/MockDataOPMDatabase.sql:1]: Looks good!

**Review by GPropersi — CHANGES_REQUESTED** [review]
Nice job! Gonna be nice to have some mock data in there.

**Review by adelhelj — APPROVED** [review]

File: database/createRecreateDatabase.py (hunk 1 above)
Comment (barreryu) [reply database/createRecreateDatabase.py:17]: We can but if we go from  "database/RecreateOPMDatabase.sql" -> "RecreateOPMDatabase.sql" it runs into a FileNotFoundError, at least on my end.

Comment (barreryu) [reply database/createRecreateDatabase.py:17]: Nevermind, I was in the wrong directory when running the file, so sorry lol

Comment (GPropersi) [reply database/createRecreateDatabase.py:17]: NP! You should be able to push a new commit, it'll be pulled into this PR

**Review by GPropersi — APPROVED** [review]

**Review by adelhelj — APPROVED** [review]

**Pull request merged**